package.json

```diff
@@ -153,6 +154,7 @@
     "html-webpack-plugin": "5.6.0",
     "jsdom": "22.1.0",
     "jsdom-global": "3.0.2",
+    "lodash": "4.17.21",
```

Comment:
I'm also interested in exploring if we could use twig to create the template instead of lodash. It would be nice to avoid introducing additional dependencies.

Reply:
For simplicity, I'd favor the template literal strings instead of requiring an additional library. There's not much complexity _yet_ to merit a separate library.

Reply:
Resolved in [d14350d](https://github.com/uswds/uswds/pull/5898/commits/d14350d0abb168b14da081af3f22be04fd587612)